cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/flow/BasicFlowletContext.java

```diff
@@ -125,7 +124,8 @@ public int getInstanceId() {
   }
 
   public LoggingContext getLoggingContext() {
-    return new FlowletLoggingContext(getNamespaceId(), getApplicationId(), getFlowId(), getFlowletId());
+    return new FlowletLoggingContext(getNamespaceId(), getApplicationId(), getFlowId(), getFlowletId(),
+                                     getRunId().getId(), String.valueOf(getInstanceId()));
```

Comment:
when we scan for logs, we would typically scan for flow + run-id ( flowlets aggregated) (at least in UI) that's the case right ? , wouldn't it be more efficient to have run-id next to flow in that scenario ?